install/empty_data.php

```diff
@@ -5161,7 +5192,14 @@ public function getEmptyData(): array
                 'content_html'             => '&lt;p&gt;##lang.change.title## : ##change.title##&lt;/p&gt;
 &lt;p&gt;##lang.change.closedate## : ##change.closedate##&lt;/p&gt;
 &lt;p&gt;##lang.satisfaction.text## &lt;a href="##change.urlsatisfaction##"&gt;##change.urlsatisfaction##&lt;/a&gt;&lt;/p&gt;',
-            ],
+            ], [
+                'id'                       => '31',
+                'notificationtemplates_id' => '31',
+                'language'                 => '',
+                'subject'                  => '##ticket.action## ##ticket.title##',
+                'content_text'             => '##ticket.reminder.bumptext##',
+                'content_html'             => '&lt;p&gt;##ticket.reminder.bumptext##&lt;/p&gt;',
```

Comment:
It should contain same values as the notification template created in `[install/migrations/update_10.0.x_to_10.1.0/notifications.php](https://github.com/glpi-project/glpi/pull/14698/files#diff-f418ecc48002a887355b5cbe867d57f0c3c0e2479c0dc00cf36c8b9e3342bd57)` file.